File: remove_variants.js
const fs = require('fs');
const glob = require('glob');
const files = glob.sync('src/**/*.tsx', { cwd: 'd:/Brandex_Projects/varsity-tribe', absolute: true });
files.forEach(file => {
  let content = fs.readFileSync(file, 'utf8');
  if (content.includes('<Container')) {
    const updated = content.replace(/<Container([^>]*)variant=['"][^'"]*['"]/g, '<Container$1');
    if (updated !== content) {
      fs.writeFileSync(file, updated);
    }
  }
});

File: remove_variants.py
import os

def replace_in_file(filepath):
    with open(filepath, 'r', encoding='utf-8') as f:
        content = f.read()
    
    if 'variant="outline"' in content:
        new_content = content.replace('variant="outline"', 'variant="secondary"')
        with open(filepath, 'w', encoding='utf-8') as f:
            f.write(new_content)
        print(f"Updated {filepath}")

for root, dirs, files in os.walk(r'd:\Brandex_Projects\varsity-tribe\src'):
    for file in files:
        if file.endswith(('.tsx', '.ts')):
            replace_in_file(os.path.join(root, file))
